physical/cassandra.go

```diff
@@ -72,6 +75,39 @@ func newCassandraBackend(conf map[string]string, logger log.Logger) (Backend, er
 	connectStart := time.Now()
 	cluster := gocql.NewCluster(hosts...)
 	cluster.Keyspace = keyspace
+
+	cluster.ProtoVersion = 2
```

Comment:
Oh I see… is it for consistency with [this](https://github.com/hashicorp/vault/blob/d06b36ce8d68a3366eb00c95a2f731891d52af79/builtin/logical/cassandra/util.go#L24)?